app/controllers/api/v1/keywords_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class KeywordsController < ApplicationController
+      def index
+        render json: KeywordSerializer.new(Keyword.new(1, 'Hello')).serializable_hash.to_json
```

Comment:
As this is an `index` action, should we implement the right format directly? 
If yes, we can just serialize a list of keywords:

```suggestion
        keywords = [Keyword.new(1, 'First keyword'), Keyword.new(1, 'Second keyword')]
        
        render json: KeywordSerializer.new(keywords).serializable_hash.to_json
```

Reply:
Another option is to use the `show` action for this story, and to keep single item serialization ;-)

Reply:
Sounds good, we can have it in the future-correct format. I will update the story.